README.md: Update installation instructions for MELPA

erikmd opened the pull request:
**Kind:** documentation

Close #222

Matafou commented on README.md:
```diff
@@ -83,7 +83,7 @@ make clean; make EMACS=/Applications/Emacs.app/Contents/MacOS/Emacs
 
-As explained in the [MELPA guide](https://melpa.org/#/getting-started), is it as easy as typing
+As explained in the [MELPA documentation](https://melpa.org/#/getting-started), it is as easy as typing
 <kbd>M-x package-list-packages RET</kbd> then `r` (**r**efresh the package list), `U` (mark **U**pgradable packages), and `x` (e**x**ecute the installs and deletions).
```
This updates all installed melpa packages. This should at least be said, and maybe a way to update only pg?

erikmd replied:
Hi @Matafou, I've just seen your remark. I will force-push a new version very soon.

erikmd commented:
I've just added the extra comments suggested by Pierre in the README.md (and I also added a commit to update the list of supported provers).

@Matafou can you take another look?
Cc @cpitclaudel

cpitclaudel commented on README.md:
```diff
@@ -24,9 +24,41 @@ Two editions of Proof General are currently available:
+Proof General, make sure you removed the old versions of Proof General
+from your Emacs context (by removing from your `.emacs` the line
+loading `PG/generic/proof-site`, or by uninstalling the proofgeneral
+package that could be provided by your OS package manager).
```
`that could be provided` → `provided`

cpitclaudel commented on README.md:
```diff
@@ -47,6 +79,27 @@ If Proof General complains about a version mismatch, make sure that the shell's
+### Using MELPA
+
+As explained in the [MELPA documentation](https://melpa.org/#/getting-started), to update all MELPA packages in one go, it is as easy as typing
+<kbd>M-x package-list-packages RET</kbd> then <kbd>r</kbd> (**r**efresh the package list), <kbd>U</kbd> (mark **U**pgradable packages), and <kbd>x</kbd> (e**x**ecute the installs and deletions).
```
`, to update all MELPA packages in one go, it is as easy as typing` → `, updating all MELPA packages in one go is as easy as typing`

cpitclaudel replied:
Will these `**` inside of a word work?

erikmd replied:
Hi @cpitclaudel, good question. On second thought it seems that only the **diff** coloring of .md files is buggy, while the GitHub rendering itself (in View mode) is very OK. So I think I could keep it as is.
Also I've done a bit of research ant it seems that the GitHub-flavored Markdown specification does allow intra-word emphasis with `*` or `**` (but obviously not `_` because of typical intra-underscores in variable identifiers). References:
- https://github.github.com/gfm/#emphasis-and-strong-emphasis
- https://github.github.com/gfm/#example-388

cpitclaudel commented on README.md:
```diff
@@ -47,6 +79,27 @@ If Proof General complains about a version mismatch, make sure that the shell's
+As explained in the [MELPA documentation](https://melpa.org/#/getting-started), to update all MELPA packages in one go, it is as easy as typing
+<kbd>M-x package-list-packages RET</kbd> then <kbd>r</kbd> (**r**efresh the package list), <kbd>U</kbd> (mark **U**pgradable packages), and <kbd>x</kbd> (e**x**ecute the installs and deletions).
+
+If you only want to update `proof-general`, you can do instead: <kbd>M-x package-list-packages RET r C-s proof-general RET i x</kbd>.
```
I'd remove this bit; it looks scary.

erikmd replied:
OK

erikmd commented:
Thanks Clément for your comments: I've taken them into account, I'll just add a MELPA badge and merge it in `master`.

cpitclaudel commented:
:+1: great job Erik.
